scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -110,6 +110,10 @@ workflow GvsExtractCallset {
                                                          else if GetNumSamplesLoaded.num_samples < 100000 then 20000 # Charlie
                                                               else 40000
 
+  Int effective_split_intervals_disk_size_override = if defined(split_intervals_disk_size_override) then select_first([split_intervals_disk_size_override])
+                                else if GetNumSamplesLoaded.num_samples < 100 then 50 # Quickstart
+                                     else 500
+
```

Comment:
Can these lines be merged?
```suggestion
Int effective_split_intervals_disk_size_override = select_first([split_intervals_disk_size_override, 
                                if GetNumSamplesLoaded.num_samples < 100 then 50 # Quickstart
                                     else 500])
```

Reply:
hmmmm yes, but then do I want to set the default as the largest and then create gates after?

Reply:
If this does what I think it does (probably good to check 🙂), it should be logically equivalent to the code you already had: if `split_interval_disk_override` is defined that value will be used, otherwise the result of the two-step step function will be used.